### PR TITLE
Docs/security policy

### PR DESCRIPTION
**Pull request by tcrosado**
## Description
Security policy required to run a bug bounty on HackerOne

## Motivation and Context
Improve contract security
Strictly define rules for vulnerability reporting

## Checklist:

- [x] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [x] Pull request has tests (at least 80% coverage)
- [x] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

**Review by GustavoRSSilva — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+## Vulnerability Disclosure policy
```
Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+<td>7.0 - 8.9</td>
+<td>&lt;= 300€</td>
+</tr><tr><td><strong>Critical</strong></td>
+<td>9.0 - 10.0</td>
+<td>&lt;= 500 €</td>
+</tr></tbody>
+</table>
+
+
+### Eligibiliy and Responsible Disclosure
```
Comment (DioPires) [inline SECURITY.md]: Should be `Eligibility`

**Review by DioPires — CHANGES_REQUESTED** [review]

**Review by DioPires — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (DioPires) [inline SECURITY.md:1]: #### Tests:
```diff 
+ 110 passing
```
#### Test coverage: 
82.44 %

**Pull request merged**